lib/colors.js

```diff
@@ -0,0 +1,95 @@
+'use strict'
+
+const fs = require('fs')
+const colorConvert = require('color-convert')
+const getImageColors = require('get-image-colors')
+const mime = require('mime-types')
+const path = require('path')
+const pickAGoodColor = require('pick-a-good-color')
+const revHash = require('rev-hash')
+const stringify = require('json-stable-stringify')
+
+/**
+ * Generates complimentary colors for an image.
```

Comment:
`compl(i|e)mentary` is not really the right term here; the color list generated by `get-image-colors` is a set of the "dominant" or "quantized" colors in an image.